tests/store/test_sqlalchemy_store.py

```diff
@@ -522,11 +533,15 @@ def test_set_tag(self):
 
         tkey = 'test tag'
         tval = 'a boogie'
+        new_val = "new val"
         tag = entities.RunTag(tkey, tval)
+        new_tag = entities.RunTag(tkey, new_val)
         self.store.set_tag(run.info.run_uuid, tag)
+        # Overwriting tags is allowed
+        self.store.set_tag(run.info.run_uuid, new_tag)
```

Comment:
Test that we can log the overwrite a tag via `set_tag` (we didn't have a direct test for this before AFAICT, though log batch tests indirectly verified this)